### DISCUSSION TITLE
Mastodon webpush is not working

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#25955] [Q&A]
I am running Mastodon v4.1.2 in a docker container and webpush notifications are not working at all despite having them enabled under my profile. I read somewhere suggesting recreating the vapid keys so I did that and still push notifications are not working. How can I troubleshoot this? Any suggestions offered would be greatly appreciated. Thank you in advance.

**Comment by vmstan** [comment]
Is your streaming server running?

**Reply by ghost** [reply]
I think I might be on to something because I have to run Mastodon behind a reverse proxy. When I bypass the reverse proxy, notifications work perfectly. So I guess I need to find a working configuration for that.
